src/NCronJob/Scheduler/QueueWorker.LogMessage.cs

```diff
@@ -39,4 +39,7 @@ internal sealed partial class QueueWorker
 
     [LoggerMessage(LogLevel.Information, "Job removed from queue: {JobType} at {RunAt}")]
     private partial void LogJobRemovedFromQueue(string jobType, DateTime? runAt);
+
+    [LoggerMessage(LogLevel.Warning, "The UseNCronJob(Async) method was not called. Startup jobs did might not be executed.")]
```

Comment:
Startup jobs did might not be executed.

"Startup jobs won't be executed" or "Startup jobs might not be executed"?

Reply:
The message will be printed basically when it is too late - therefore I did use the "past tense"

Reply:
Ok. But I think "Startup jobs did might not be executed." may not be proper English :wink:

Maybe "Startup jobs might not have been executed"?

Reply:
That sounds better (Y) I'll take that.